src/main/java/uk/gov/hmcts/reform/preapi/controllers/AuditController.java

```diff
@@ -47,4 +58,25 @@ public ResponseEntity<Void> upsertAudit(@RequestHeader HttpHeaders headers,
         this.auditService.upsert(createAuditDTO, userId);
         return new ResponseEntity<>(HttpStatus.CREATED);
     }
+
+    @GetMapping
+    @Operation(operationId = "getAuditLogs", summary = "Search all Audits")
+    @PreAuthorize("hasAnyRole('ROLE_SUPER_USER', 'ROLE_LEVEL_1', 'ROLE_LEVEL_2', 'ROLE_LEVEL_3', 'ROLE_LEVEL_4')")
```

Comment:
This endpoint should only be exposed to Super users
```suggestion
    @PreAuthorize("hasAnyRole('ROLE_SUPER_USER')")
```